### ISSUE TITLE
The input size in the code for the MINC data set is 224 instead of 352 in the paper. Is the code for single-scale training by default? Besides, how can I switch the multi-scale training mode. Looking forward to your answer.

### ISSUE THREAD

**Issue by roseif** [zhanghang1989/PyTorch-Encoding#340]

**Comment by zhanghang1989** [comment]
The original implementation was using Torch at https://github.com/zhanghang1989/Torch-Encoding-Layer

The PyTorch implementation has slightly different settings.

**Comment by zhanghang1989** [comment]
You can use `-multisize true`, the full command is

```
 th main.lua -retrain resnet-50.t7 -ft true \
 -netType encoding -nCodes 32 -dataset minc \
 -data minc/ -nClasses 23 -batchSize 64 \
 -nGPU 4 -multisize true
```

**Comment by roseif** [comment]
谢谢
